src/Math-Complex/PMComplex.class.st

```diff
@@ -390,7 +390,7 @@ PMComplex >> arcTan: denominator [
 			res]
 ]
 
-{ #category : #arithmetic }
+{ #category : #'polar coordinates' }
```

Comment:
I think this is worth doing because 

`z = r exp(I theta)`

is the polar coordinate form which is taught as a concept explicitly. I feel the code could reflect that too,